src/shared/components/custom-input/input.component.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+import { InputPropsTypes } from '@/shared/types/input.types'
+
+const CustomInput: FC<InputPropsTypes> = ({
+    label,
+    type,
+    placeholder,
+    required,
+    errorMessage,
+    name,
+    defaultValue,
+    className,
+    changeEvent,
+    readonly,
+}) => {
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        changeEvent?.(e)
+    }
+    return (
+        <div>
```

Comment:
No styling for this???

Reply:
done it by making flex container and flex direction is colum